src/haz3lschool/Gradescope.re

```diff
@@ -67,7 +67,13 @@ module Main = {
     };
     let model_results = ModelResults.init(spliced_elabs(exercise));
     let stitched_dynamics = stitch_dynamic(exercise, Some(model_results));
-    let grading_report = exercise.eds |> GradingReport.mk(~stitched_dynamics);
+    let syntax_report = syntax_test(exercise);
+    let grading_report =
+      exercise.eds
+      |> GradingReport.mk(
```

Comment:
the syntax report should be generated when the GradingReport is made, not separately in Gradescope